### PR TITLE
Added twitter timeline

### PR DESCRIPTION
**Pull request by ErikUu**
Added twitter tweet timeline for _#open2017_ - closes issue #11

### CODE REVIEW

**Comment by raisedadead** [comment]
@ErikUu Awesome, can we adjust the width dynamically, for instance on my screen, looks a bit off.

image

File: index.html [hunk 1]
```diff
@@ -67,6 +69,13 @@ <h1 class='ginormous-text'>#Open2017</h1>
 				</div>
 			</div>
 
+			
+			<div class="doc-section pad" id="twitter-timeline">
+				<a id="twitter-timeline" class="twitter-timeline"  href="https://twitter.com/hashtag/Open2017" data-widget-id="811886173450891264">#Open2017 Tweets</a>
+            	<script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+"://platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs");</script>
```
Comment (atjonathan) [inline index.html]: The indentation needs fixed on this line ❗️

File: index.html [hunk 2]
```diff
@@ -67,6 +69,16 @@ <h1 class='ginormous-text'>#Open2017</h1>
...
 			</div>
 
+			
+			<div class="doc-section pad" id="twitter-timeline">
+				<a id="twitter-timeline"
+					class="twitter-timeline" 
+					href="https://twitter.com/hashtag/Open2017" 
+					data-widget-id="811886173450891264">#Open2017 Tweets
+				</a>
+			</div>.
```
Comment (atjonathan) [inline index.html]: There shouldn't be a dot here.

Comment (ErikUu) [inline index.html]: Good catch, how messy of me.

**Comment by QuincyLarson** [comment]
@ErikUu Nice work! I've resolved the merge conflicts and merged this.

**Pull request merged**